### ISSUE TITLE
[BUG] UBL with PROBE_MANUALLY doesn't lower to MANUAL_PROBE_START_Z and stays at Z_CLEARANCE_BETWEEN_PROBES

### ISSUE THREAD

**Issue by qurvento** [MarlinFirmware/Marlin#19795]
### Bug Description

UBL with defined PROBE_MANUALLY doesn't move to MANUAL_PROBE_START_Z and stays at Z_CLEARANCE_BETWEEN_PROBES height when manually building a mesh via LCD.

### My Configurations

Creality Ender-3 v4.2.2
Marlin 2.0.7.1
AUTO_BED_LEVELING_UBL, PROBE_MANUALLY, LCD_BED_LEVELING
[cfg.zip](https://github.com/MarlinFirmware/Marlin/files/5397588/cfg.zip)

### Steps to Reproduce

1. Select "Motion/Unified Bed Leveling/UBL Tools/Manually Build Mesh".

**Expected behavior:** moves to center and then to Z = MANUAL_PROBE_START_Z

**Actual behavior:** moves to center and stays at Z = Z_CLEARANCE_BETWEEN_PROBES

In `unified_bed_leveling::G29()` Z_CLEARANCE_BETWEEN_PROBES could be changed to MANUAL_PROBE_START_Z, if defined.
```
              g29_card_thickness = parser.has_value() ? parser.value_float() : measure_business_card_thickness(float(Z_CLEARANCE_BETWEEN_PROBES));
```
and
```
            const float height = parser.floatval('H', Z_CLEARANCE_BETWEEN_PROBES);
            manually_probe_remaining_mesh(g29_pos, height, g29_card_thickness, parser.seen('T'));
```

**Comment by boelle** [comment]
is this related to #19814 ?

**Comment by qurvento** [comment]
no

**Comment by sjasonsmith** [comment]
I have posted a PR to resolve this. I don't know yet whether it will be accepted and merged. Please feel free to test it and comment on your results in the PR.

**Comment by qurvento** [comment]
Thank you.

**Comment by boelle** [comment]
> Thank you.

did you test it?

**Comment by sjasonsmith** [comment]
This change was merged into the bugfix-2.0.x branch today. I'm closing this issue out, go ahead and reply if you encounter problems using the change.
